Documentation/quickstart.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
this file was just renamed, right?

Reply:
Yes, I renamed the file back from `quickstart-toc.md` to `quickstart.md`.
I have some plans to make it "more" interesting in the third phase of my doc rework and improvement plan.